src/main/java/net/dv8tion/jda/api/interactions/commands/build/CommandData.java

```diff
@@ -146,6 +146,18 @@ public interface CommandData extends SerializableData
     @Nonnull
     CommandData setGuildOnly(boolean guildOnly);
 
+    /**
+     * Sets whether this command should only be usable in NSFW (age-restricted) channels.
+     * <br>Default: false
+     *
+     * @param  nsfw
+     *         True, to make this command nsfw
+     *
+     * @return The builder instance, for chaining
+     */
+    @Nonnull
+    CommandData setNSFW(boolean nsfw);
```

Comment:
Should we specify that you have to explicitly enable nsfw commands in the client in order to see them?

Reply:
Note that this is only in the case of DM commands